package.json

```diff
@@ -99,5 +99,8 @@
     "stylelint": "^7.3.1",
     "stylelint-config-standard": "^13.0.2",
     "through": "^2.3.8"
+  },
+  "dependencies": {
+    "mousetrap": "^1.6.0"
```

Comment:
Well, since only a dev should run `npm install`, it doesn't matter much I guess.